### ISSUE TITLE
Crash in tellcore_tool --list-devices

### ISSUE THREAD

**Issue by mkaut** [erijo/tellcore-py#4]
I have just installed tellcore-py on a debian (wheezy) system, with Python 3.2 (using pip-3.2).

Running `tellcore_tool --list-devices` fails with the following error message:
```
Traceback (most recent call last):
  File "/usr/local/bin/tellcore_tool", line 133, in <module>
    print_devices(core.devices())
  File "/usr/local/bin/tellcore_tool", line 43, in print_devices
    for name, value in device.parameters().iteritems():
AttributeError: 'dict' object has no attribute 'iteritems'
```
Is this a genuine bug, or am I just missing something?

**Comment by erijo** [comment]
Thanks for the report. I've just released v1.1.1 where the problem should be fixed.

**Comment by mkaut** [comment]
Thanks, that was quick :-)
I can confirm that the new version works for me as well.
